docs/contributors/mocking.md

```diff
@@ -0,0 +1,8 @@
+### Mocking in minikube
+
+To mock interfaces in unit tests use [testify/mock](https://github.com/stretchr/testify) based mocks. 
+The mocks can be generated using mockery. 
+We have a wrapper script to update or create a new mock: `hack/gen_mocks.sh`.
```

Comment:
Is the idea that we don't edit the mocks and they're always generated with this?

Reply:
yes!

Reply:
removed mocking.